app.rb

```diff
@@ -0,0 +1,66 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './rental'
+require_relative './book'
+
+class App
+  attr_reader :book_list, :people_list
+
+  def initialize
+    @book_list = []
+    @people_list = []
+    @rental_list = []
+  end
+
+  def add_book(title, author)
+    Book.new(title, author)
+  end
+
+  def add_student(classroom, age, name, parent_permission)
+    student = Student.new(classroom, age, name, parent_permission)
+    @people_list.push({ value: student, type: 'Student' })
+  end
+
+  def add_teacher(specialization, age, name)
+    teacher = Teacher.new(specialization, age, name)
+    @people_list.push({ value: teacher, type: 'Teacher' })
+  end
+
+  def add_rental(date, book_num, person_num)
+    book = @book_list[book_num - 1]
+    person = @people_list[person_num - 1][:value]
+    rental = Rental.new(date, book, person)
+    @rental_list.push(rental)
+  end
+
+  def display_books
```

Comment:
The user is not able to see the book list when clicking on option `1`. Kindly make sure the book list is displayed.